### PR TITLE
Satellite Imagery Bucket 

### PR DESCRIPTION
**Pull request by carolinepacheco**
Updated Satellite Imagery Bucket documentation.

### CODE REVIEW

**Review by andrewssobral — APPROVED** [review]

File: src/docs/PML/PMLUserGuide.adoc [hunk 1]
```diff
@@ -4714,9 +4715,24 @@ image::Fetch_Satellite_Images.png[align=center]
...
+| Downloads files or folders to the data space.
+| String (default=PA:GLOBAL_FILE)
+| `TIME_TO_RETRIEVE_IN_SECONDS`
+| Defines time in second to request an offline product from the Long Term Archive (LTA) in the Copernicus Open Access Hub’s.
+| Int (default=900 seconds)
+| `TIME_TO_CHECK_ONLINE_IN_SECONDS`
+| Defines time in second to check if a product is online.
+| Int (default=1800 seconds)
+| `WALLTIME`
+|  Determines the maximum execution time of a task
```
Comment (Hiba-Alili) [inline src/docs/PML/PMLUserGuide.adoc:4728]: extra space before Determines the maximum and a . is missed by the end of the sentence

File: src/docs/PML/PMLUserGuide.adoc [hunk 2]
```diff
@@ -4714,9 +4715,24 @@ image::Fetch_Satellite_Images.png[align=center]
 | `SEARCH_ENGINE`
 | Defines an engine to search and download satellite images.
 | List [Peps, Copernicus, All] (default=Copernicus)
+| `IMPORT_FROM`
+| Downloads files or folders to the data space.
+| String (default=PA:GLOBAL_FILE)
+| `TIME_TO_RETRIEVE_IN_SECONDS`
+| Defines time in second to request an offline product from the Long Term Archive (LTA) in the Copernicus Open Access Hub’s.
```
Comment (Hiba-Alili) [inline src/docs/PML/PMLUserGuide.adoc:4722]: time in second**s**

File: src/docs/PML/PMLUserGuide.adoc [hunk 3]
```diff
@@ -4714,9 +4715,24 @@ image::Fetch_Satellite_Images.png[align=center]
...
+| Int (default=900 seconds)
+| `TIME_TO_CHECK_ONLINE_IN_SECONDS`
+| Defines time in second to check if a product is online.
```
Comment (Hiba-Alili) [inline src/docs/PML/PMLUserGuide.adoc:4725]: the same as above (in second**s**)

File: src/docs/PML/PMLUserGuide.adoc [hunk 4]
```diff
@@ -4725,6 +4741,7 @@ NOTE: Please click https://peps.cnes.fr/rocket/#/register[here^] to create a new
 
 NOTE: Please click https://scihub.copernicus.eu/dhus/#/self-registration[here^] to create a new user account from Copernicus website.
 
+WARNING: Note que as variaveis `TIME_TO_RETRIEVE_IN_SECONDS`, `TIME_TO_CHECK_ONLINE_IN_SECONDS` and `WALLTIME` are valid only for the Copernicus platform.
```
Comment (Hiba-Alili) [inline src/docs/PML/PMLUserGuide.adoc:4744]: Note that

**Pull request merged**